parachain/runtime/rococo-common/src/lib.rs

```diff
@@ -8,6 +8,8 @@
 use frame_support::parameter_types;
 use xcm::opaque::lts::NetworkId;
 
+pub const INBOUND_QUEUE_MESSAGES_PALLET_INDEX: u8 = 80;
```

Comment:
Looks like there is an API for retrieving a pallet's index: https://substrate.stackexchange.com/a/794

Reply:
Ah ok makes sense 👌